khmer/khmer_args.py

```diff
@@ -389,18 +393,22 @@ def add_loadgraph_args(parser):
                         help='load a precomputed k-mer graph from disk')
 
 
-def calculate_graphsize(args, graphtype, multiplier=1.0):
+def calculate_graphsize(args, graphtype, multiplier=1.0, _sysmem=None):
```

Comment:
Yes, it's solely for the unit tests.

By "attract attention", do you mean *bad* attention? I thought the leading underscore was a pretty common convention for saying "this is internal, you shouldn't mess with this unless you know what you're doing."